app/Http/Client/Alchemy/AlchemyPendingRequest.php

```diff
@@ -395,11 +396,20 @@ public function parseNft(array $nft, int $networkId, bool $convertTokenNumber =
 
         $tokenNumber = $convertTokenNumber === true ? CryptoUtils::hexToBigIntStr($nft['id']['tokenId']) : $nft['id']['tokenId'];
 
+        $error = Arr::get($nft, 'error');
+        if (! empty($error)) {
+            Log::info('AlchemyPendingRequest: Filter NFT', [
```

Comment:
Can you include the collection address, in case it's difficult to infer it from the name alone (due to ambiguity, etc.)